filters/include/pcl/filters/uniform_sampling.h

```diff
@@ -72,7 +72,8 @@ namespace pcl
       typedef boost::shared_ptr<const UniformSampling<PointT> > ConstPtr;
 
       /** \brief Empty constructor. */
-      UniformSampling () :
+      UniformSampling (bool extract_removed_indices = false) :
```

Comment:
We will need to remember to delete it after ^^.

Let's think the other way around, maybe 1.8.1 simply doesn't make sense anymore, it's been more than year. Let's just adopt the new 1.8.0-dev designation and start working on a release with new features, etc... Under that context it's no problem to have the ABI break.